### PR TITLE
[bugfix] Test tls sfeatures

### PR DESCRIPTION
**Pull request by ehsanziyaee**
remove some redundant component from Cmake

### CODE REVIEW

File: SpotifyInterface.c [hunk 1]
```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
             break;
 
         case GetUserInfo:
+        //SEZ@Done ESP_LOGW just for test.
+        ESP_LOGW(TAG, "GetUserInfo");
             Spotify_GetInfo(Command, PrivateHandler.token.AccessToken);
-            IsSuccessfull = PrivateHandler.SpotifyBuffer.status == 200;
+        ESP_LOGW(TAG, "Spotify_GetInfo");    
+            IsSuccessfull = (PrivateHandler.SpotifyBuffer.status == 200)? true:false;
```
Comment (Behn4m) [inline SpotifyInterface.c]: what is the reason for this change? previous code also had same result
if the `PrivateHandler.SpotifyBuffer.status` is equal to `200`, retun `true`, if not retun `false`

File: SpotifyInterface.c [hunk 2]
```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
             break;
 
         case GetUserInfo:
+        //SEZ@Done ESP_LOGW just for test.
+        ESP_LOGW(TAG, "GetUserInfo");
```
Comment (Behn4m) [inline SpotifyInterface.c]: you want to keep this log? I think we print them later on top layer

File: SpotifyInterface.c [hunk 3]
```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
...
+        ESP_LOGW(TAG, "Spotify_GetInfo");    
+            IsSuccessfull = (PrivateHandler.SpotifyBuffer.status == 200)? true:false;
             if (!IsSuccessfull)
             {
                 ESP_LOGW(TAG, "No user is found");
                 retValue = false;
                 break;
             }
-            ExtractUserInfoParamsfromJson(PrivateHandler.SpotifyBuffer.MessageBuffer, InterfaceHandler->UserInfo);
+        ESP_LOGW(TAG, "IsSuccessfull");
```
Comment (Behn4m) [inline SpotifyInterface.c]: same thing here, we are showing the status later on higher layer

File: SpotifyInterface.c [hunk 4]
```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
...
             {
                 ESP_LOGW(TAG, "No user is found");
                 retValue = false;
                 break;
             }
-            ExtractUserInfoParamsfromJson(PrivateHandler.SpotifyBuffer.MessageBuffer, InterfaceHandler->UserInfo);
+        ESP_LOGW(TAG, "IsSuccessfull");    
+            ExtractUserInfoParamsfromJson(PrivateHandler.SpotifyBuffer.MessageBuffer, 
+            InterfaceHandler->UserInfo);
+            ESP_LOGW(TAG, "ExtractUserInfoParamsfromJson");
```
Comment (Behn4m) [inline SpotifyInterface.c]: and same here. if the status is going to be reported later, this one is not needed. Maybe they could be `ESP_LOGI` or `ESP_LOGD` to be visibale only during debug

**Review by Behn4m — APPROVED** [review]
I placed some comments. the code is ok, just make sure log are handled to be minimal and keep code as simple as possible

**Review by Behn4m — APPROVED** [review]
Good PR. just make sure the logs are handled to be minimal and the keep code simple

File: SpotifyInterface.c (hunk 1 above)
Comment (ehsanziyaee) [reply SpotifyInterface.c]: this is an example for right Syntax of Conditional/Ternary Operator in C/C++. but I change it just for one one your cases

File: SpotifyInterface.c (hunk 2 above)
Comment (ehsanziyaee) [reply SpotifyInterface.c]: i want to show the other, this is the place of an error occurring, but I can remove them

File: SpotifyInterface.c (hunk 1 above)
Comment (Behn4m) [reply SpotifyInterface.c:353]: Screenshot 2024-06-26 104938
🙄

Comment (Behn4m) [reply SpotifyInterface.c]: image
🙄😊
